modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -260,14 +265,20 @@ void testDropZoneDoNotPropagateDataNodesAfterScaleUp() throws Exception {
 
         assertDataNodesForZone(1, clusterNodesNames2, keyValueStorage);
 
-        assertZoneScaleUpChangeTriggerKey(1, 1, keyValueStorage);
+        assertNotNull(keyValueStorage.get(zoneDataNodesKey(1).bytes()).value(),
```

Comment:
Seems that we already checked exact equality of data nodes. See line above. So that NotNull check check dataNodes is useless.

Reply:
Fixed.